docs/api.rst

```diff
@@ -8106,3 +8107,61 @@ ERC20 token info
    :statuscode 200: No critical error found.
    :statuscode 409: There is an error with the address.
    :statuscode 500: Internal rotki error.
+
+All Binance markets
+======================
+
+.. http:get:: /api/(version)/exchanges/binance/pairs
+
+   Doing a GET to this endpoint will return the market pairs available at binance.
+
+   .. note::
+      This endpoint will only return information if either Binance or Binance US are configured
+
+   **Example Request**:
+
+   .. http:example:: curl wget httpie python-requests
+
+      GET /api/1/exchanges/binance/pairs HTTP/1.1
+      Host: localhost:5042
+
+   **Example Response**:
+
+   .. sourcecode:: http
+
+      HTTP/1.1 200 OK
+      Content-Type: application/json
+
+      {
+          "result": ["BTCUSD", "ETHUSD", "XRPUSD"],
+          "message": ""
+      }
+
+User selected Binance markets
+================================
+
+.. http:get:: /api/(version)/exchanges/binance/pairs/(exchange account name)
+
+   Doing a GET to this endpoint will return the market pairs that the user has selected to be queried at binance.
+
+   .. note::
```

Comment:
I think this note is not needed here. Having an exchange account name as a requirement implies that.